explain-partitions.md

```diff
@@ -0,0 +1,132 @@
+---
+title: 用 EXPLAIN 查看分区的 SQL 执行计划
```

Comment:
@yikeke I don't think `使用分区来解释` is correct, because `EXPLAIN` is used to explain the execution plan of SQL statements that use partitions.

Reply:
@TomShawn I see what you mean. The **linguistic ambiguity** of the original title "Explain Statements Using Partitions" got me confused. Please help confirm with the author to refine the title, both in Chinese and English. Thanks. @Joyinqin

Reply:
Thanks for comments~@TomShawn @yikeke

Reply:
So, what does the author say? @Joyinqin

Reply:
It seems no problem to keep the original title in Chinese. I have drafted a PR to modify the titles of all English versions of Morgan's PRs to avoid language ambiguity.@yikeke

Reply:
```suggestion
title: 用 EXPLAIN 查看 SQL 语句需要访问的分区
```
"分区的 SQL 执行计划" sounds weird to me. I made a suggestion as above. PTAL @imtbkcat

Reply:
LGTM @yikeke